src/components/Shirts/shirts.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import "./shirts.css";
+import shirtsList from  "../../shirtsList.json";
+import Shirt from "../Shirt/shirt";
+
+class Shirts extends Component {
+ getShirts(){
+   const renderedShirts = shirtsList.map((shirt)=>{
```

Comment:
Destructuring would be great here e.g. `({imgUrl, label, price})` to avoid repeating "shirt".